.github/workflows/rust.yml

```diff
@@ -2,9 +2,11 @@ name: Rust
 
 on:
   push:
-    branches: [master, main]
+    branches:
+      - main
   pull_request:
-    branches: [master, main]
+    paths:
```

Comment:
@jkilpatr is it fine to only run rust tests when the files are touched? Want to make sure these tests are dependent on things in the module directory.